src/pools/abstract-pool.ts

```diff
@@ -399,4 +424,10 @@ export abstract class AbstractPool<
       }
     }
   }
+
+  private checkEmitBusy (): void {
+    if (this.isPoolBusy()) {
```

Comment:
Do you thing that we should use a getter also for `isPoolBusy`? Like as for `isDynamic`?